Dell/recovery_backend.py

```diff
@@ -970,6 +1012,13 @@ def create_ubuntu(self, recovery, revision, iso, platform, no_update, sender=Non
             xorrisoargs.append('boot/efi.img')
             xorrisoargs.append('-no-emul-boot')
             xorrisoargs.append('-isohybrid-gpt-basdat')
+        elif os.path.exists(os.path.join('tmp', 'backingfile-efi.img')):
```

Comment:
as mentioned above, please generate temporary directory/filename to use for this (or use one of them already generated).

Reply:
In the 2nd patch changed the file path to leverage where the `efi.img` is expected to be seen i.e.: `/tmp/tmp{random}/boot/efi.img`